app/graphql/mutations/rock_and_pebbles/activate_rock_pebble_relationship.rb

```diff
@@ -0,0 +1,23 @@
+module Mutations
+  module RockAndPebbles
+    class ActivateRockPebbleRelationship < ::Mutations::BaseMutation
+      argument :id, ID, required: true
+      type Types::RockAndPebbleType
+
+      def resolve(attributes)
+        rock_and_pebble = RockAndPebble.find(attributes[:id])
+        send_pebble_confirmation_email(rock_and_pebble)
+        rock_and_pebble.update(active: true)
```

Comment:
I forget what we discussed...did we eventually want to use this same mutation if someone wants to end their relationship? If so the rock_and_pebble.toggle!(:active) might be useful. If we want a separate mutation for that then this looks great!